client/src/components/Navbar/LoggedIn/LoggedIn.tsx

```diff
@@ -0,0 +1,58 @@
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Avatar from '@material-ui/core/Avatar';
+import Badge from '@material-ui/core/Badge';
+import Box from '@material-ui/core/Box';
+import IconButton from '@material-ui/core/IconButton';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import Link from '@material-ui/core/Link';
+import React from 'react';
+import { useAuth } from '../../../context/useAuthContext';
+
+import useStyles from './useStyles';
+
+export default function LoggedIn(): JSX.Element {
+  const classes = useStyles();
+
+  const { logout } = useAuth();
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleLogout = () => {
+    handleClose();
+    logout();
+  };
+
+  return (
+    <React.Fragment>
+      <Box mr={8}>
+        <Link href="/dashboard/my-sitters" color="inherit">
```

Comment:
@simalicrum You may have missed this! Please use react-router's Link and not the one from MUI :)

Reply:
Fixed